core/src/main/resources/jenkins/management/Messages.properties

```diff
@@ -39,7 +39,7 @@ SystemInfoLink.DisplayName=System Information
 SystemInfoLink.Description=Displays various environmental information to assist trouble-shooting.
 
 SystemLogLink.DisplayName=System Log
-SystemLogLink.Description=System log captures output from <tt>java.util.logging</tt> output related to Jenkins.
+SystemLogLink.Description=System log captures output from <code>java.util.logging</code> output related to Jenkins.
```

Comment:
Unrelated but `<tt>` is a [obsolete](https://developer.mozilla.org/en/docs/Web/HTML/Element/tt) element. Oldschool 😄